urbanairship-react-native/ios/UARCTModule/UrbanAirshipReactModule.m

```diff
@@ -178,14 +178,20 @@ - (instancetype)init{
 RCT_REMAP_METHOD(isUserNotificationsOptedIn,
                  isUserNotificationsOptedIn_resolver:(RCTPromiseResolveBlock)resolve
                  rejecter:(RCTPromiseRejectBlock)reject) {
+    BOOL optedIn = [[UAirship push] userPushNotificationsEnabled];
```

Comment:
you actually need to recreate the logic in this method - https://github.com/urbanairship/ios-library/blob/main/Airship/AirshipCore/Source/Internal/UAPush.m#L652

Reply:
I don't understand why I wasn't able to get "isRegisteredForRemoteNotifications" and the privacyManager from UAPush, so I haven't included them in the logic

Reply:
I'm not so worried about isRegisteredForRemoteNotifications, but unless this includes privacy manager the method doesn't mean the same thing it does on Android. I believe you can get the shared privacy manager from UAirship